chia/_tests/wallet/conftest.py

```diff
@@ -197,3 +209,89 @@ async def wallet_environments(
                 ],
                 tx_config,
             )
+
+            if as_vault:
```

Comment:
Can we move this to a free function just to keep the body of _this_ function focused?